application/aam-external-mock-service/src/main/kotlin/com/aamdigital/aamexternalmockservice/Application.kt

```diff
@@ -0,0 +1,28 @@
+package com.aamdigital.aamexternalmockservice
+
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan
+import org.springframework.boot.runApplication
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RestController
+import java.util.*
+
+@SpringBootApplication
+@ConfigurationPropertiesScan
+@EnableJpaRepositories
+class Application
+
+fun main(args: Array<String>) {
+  TimeZone.setDefault(TimeZone.getTimeZone("UTC"))
+  runApplication<Application>(*args)
+}
+
+@RestController
+class HelloFriendController {
```

Comment:
I am not sure where we can document this best but let's add some general code comment or readme briefly explaining what the purpose of aam-external-mock-service is. (possibly in a completly different file or folder also)